xmake.lua

```diff
@@ -103,6 +103,9 @@ local modules = {
 				add_defines("VK_USE_PLATFORM_WAYLAND_KHR")
 			elseif is_plat("macosx") then
 				add_defines("VK_USE_PLATFORM_METAL_EXT")
+				add_links("objc")
```

Comment:
Prefer `add_syslinks` for system-specific links

Reply:
It was needed when I was using the objc runtime directly but it doesn't seem necessary now. I guess xmake can pull it automatically since there is a objc file linked. I'll remove.